build.sbt

```diff
@@ -122,6 +123,11 @@ lazy val httpMarshallersJava = project("akka-http-marshallers-java")
 lazy val httpJackson =
   httpMarshallersJavaSubproject("jackson")
 
+lazy val httpCaching = project("akka-http-caching")
```

Comment:
I am wondering if it would be better to put the core caching API in the `akka-http` project and then have the Caffeine related code in `akka-http-caching/akka-http-caffeine` similar to how we manage marshallers with external dependencies. This could make caching more self-documenting.

```scala
import akka.http.caching.caffeine.CaffeineCachingSupport.routeCache

val route =
  cache(routeCache(), simpleKeyer) {
    // ...
  }
```

Reply:
I think it's good for now. We can think about moving it in later on.